src/onConsentChange.ts

```diff
@@ -19,7 +20,10 @@ const invokeCallback = (callback: CallbackQueueItem, state: ConsentState) => {
 
 const getConsentState: () => Promise<ConsentState> = async () => {
 	if (window.__uspapi) {
-		// in USA - https://git.io/JUOdq
+		// in USA or AUS - https://git.io/JUOdq
+		if (window.guCmpHotFix.framework === 'aus')
```

Comment:
Any thoughts on this?

Reply:
should we introduced a singleton that stores the active framework?

Reply:
That would make sense!

Reply:
something like 

```ts
// currentFramework.ts
let currentFramework: Framework?;

export const getCurrentFramework= () => currentFramework;
export const setCurrentFramework = (framework: Framework) => currentFramework = framework;
```
then we can import it anywhere in the CMP, and set it during `init`

what do you think?

Reply:
discussed irl